### PR TITLE
Add replaceable tags to constants

### PR DESCRIPTION
**Pull request by haszi**
This PR adds `<replaceable>` tags to constants that are already using a `<constant>` tag and an `*` to mark the replaceable part of the constant.

### CODE REVIEW

File: reference/oauth/oauth/fetch.xml [hunk 1]
```diff
@@ -45,7 +45,7 @@
      <term><parameter>http_method</parameter></term>
      <listitem>
       <para>
-       One of the <constant>OAUTH_HTTP_METHOD_*</constant>
+       One of the <constant>OAUTH_HTTP_METHOD_<replaceable>*</replaceable></constant>
        <link linkend="oauth.constants">OAUTH constants</link>, which includes
        GET, POST, PUT, HEAD, or DELETE.
```
Comment (Girgias) [inline reference/oauth/oauth/fetch.xml:50]: Possibly rewrite this to remove the `<link>` tag?

File: reference/oauth/oauthprovider/checkoauthrequest.xml [hunk 2]
```diff
@@ -38,7 +38,7 @@
     <listitem>
      <para>
       The <acronym>HTTP</acronym> method. Optionally pass in one of the 
-      <constant>OAUTH_HTTP_METHOD_*</constant> <link linkend="oauth.constants">OAuth constants</link>.
```
Comment (Girgias) [inline reference/oauth/oauthprovider/checkoauthrequest.xml:41]: Ditto

File: reference/pdo_pgsql/PDO/pgsqlGetNotify.xml [hunk 3]
```diff
@@ -26,7 +26,7 @@
      <listitem>
       <para>
        The format the result set should be returned as, represented as a
-       <link linkend="pdostatement.fetch"><constant>PDO::FETCH_*</constant></link>
```
Comment (Girgias) [inline reference/pdo_pgsql/PDO/pgsqlGetNotify.xml:29]: Remove the outer `<link>` tag.

File: reference/geoip/functions/geoip-database-info.xml [hunk 4]
```diff
@@ -33,7 +33,7 @@
       <para>
        The database type as an integer. You can use the
        <link linkend="geoip.constants">various constants</link> defined with
-       this extension (ie: GEOIP_*_EDITION).
+       this extension (ie: GEOIP_<replaceable>*</replaceable>_EDITION).
       </para>
```
Comment (Girgias) [inline reference/geoip/functions/geoip-database-info.xml:37]: This feels like it needs to be rewritten and in any case it needs the `<constant>` tag.

**Review by Girgias — CHANGES_REQUESTED** [review]
I would drop the GeoIP changes from this PR as they require more detailed changes.

**Comment by haszi** [comment]
As requested, I removed the GeoIP changes and the `<link>`s.

**Review by Girgias — APPROVED** [review]

**Pull request merged**